bridge/test/BridgeV3_to_bridgeV4_test.js

```diff
@@ -75,7 +75,7 @@ contract('Bridge Multichain Deploy Check', async function (accounts) {
       const bridgeV4Implementation = new web3.eth.Contract(this.bridgeV4.abi, bridgeProxy.address);
 
       result = await bridgeV4Implementation.methods.version().call();
-      assert.equal(result, 'v4');
+      assert.equal(result, 'v3');
```

Comment:
I don't understand this change, if its v4 why is v3 expected?

Reply:
There is no V4 for bridge, this was wrong in the tests. I don't know if they planed to do it in the past but there is not Bridge V4